CoreUtils/src/main/java/com/raizlabs/android/coreutils/app/FragmentStackManagerFragment.java

```diff
@@ -0,0 +1,372 @@
+package com.raizlabs.android.coreutils.app;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentTransaction;
+
+public class FragmentStackManagerFragment extends Fragment {
+
+    /*
```

Comment:
This doesn't belong in the public class description because it's implementation details that shouldn't be exposed to the consumer.